pkg/action/sync.go

```diff
@@ -42,29 +37,44 @@ func (s *Action) sync(ctx context.Context, c *cli.Context, store string) error {
 				continue
 			}
 		}
-
 		numMPs++
-		_ = s.syncMount(ctx, mp)
+		if syncResult, _ := s.syncMount(ctx, mp); syncResult != nil {
+			changes[syncResult.name] = syncResult.changed
+		}
 	}
 	out.Green(ctx, "All done")
 
 	// calculate number of changes entries
-	if l, err := s.Store.Tree(ctx); err == nil {
-		numEntries = len(l.List(0)) - numEntries
+	totalAdded := 0
+	totalRemoved := 0
+	if len(changes) > 0 {
+		out.Print(ctx, "Sync summary:\n")
+	}
+	for k := range changes {
+		if changes[k] > 0 {
+			out.Print(ctx, fmt.Sprintf("Mount %s: Added %d entries", k, changes[k]))
+			totalAdded += changes[k]
+		} else if changes[k] < 0 {
+			out.Print(ctx, fmt.Sprintf("Mount %s: Removed %d entries", k, -1*changes[k]))
+			totalRemoved += -1 * changes[k]
```

Comment:
Remove the ´-1 *´, just add up what's removed.

Reply:
Since if entries were removed it returns negative number it's necessary to not print "removed -5 entries". Or I am wrong?